benchmarks/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ macro(fiber_add_benchmark fiber_benchmark)
         target_link_libraries(${fiber_benchmark} ffte)
     endif()
     if (FIBER_ENABLE_SWFFT)
-        target_link_libraries(${fiber_benchmark} swfft)
+        target_link_libraries(${fiber_benchmark} fiber_swfft)
```

Comment:
We should keep original library name: SWFFT

Reply:
There is no "original library".  See above.